plugins/out_es/es.c

```diff
@@ -159,14 +163,53 @@ static inline int es_pack_map_content(msgpack_packer *tmp_pck,
             msgpack_pack_map(tmp_pck, v->via.map.size);
             es_pack_map_content(tmp_pck, *v, ctx);
         }
+        /*
+         * The value can be any data type, if it's an array we need to
+         * pass it to es_pack_array_content.
+         */
+        else if (v->type == MSGPACK_OBJECT_ARRAY)
```

Comment:
fix brace: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#braces-usage-on-conditionals-loops-and-functions